syntax = "proto3";

package test;

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

message NumberRequest {
  int32 count = 1;
}

message NumberReply {
  int32 number = 1;
}

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc StreamNumbers (NumberRequest) returns (stream NumberReply) {}
}